go.mod

```
@@ -343,3 +343,7 @@ replace github.com/go-chi/chi/v4 => github.com/go-chi/chi v4.0.0+incompatible
 
 // see https://github.com/ionos-cloud/sdk-go/issues/27
 exclude github.com/ionos-cloud/sdk-go/v6 v6.0.5851
+
+// This replace can be removed once it is patched to v0.3.2 upstream
+// see https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/exporter/datadogexporter/go.mod#L116
+replace github.com/theupdateframework/go-tuf => github.com/DataDog/go-tuf v0.3.0--fix-localmeta
```

```suggestion
replace github.com/theupdateframework/go-tuf v0.3.1 => github.com/DataDog/go-tuf v0.3.0--fix-localmeta
```

Can we specify the source version, too?  That way we should be in the clear when DataDog update to `>=v0.3.2` and can then remove this at our leisure.